### ISSUE TITLE
[BUG] - command 'bookmarks.toggle' not found error in V13.0

### ISSUE THREAD

**Issue by kalahiri** [alefragnani/vscode-bookmarks#387]
**Environment/version**
Windows 

- Extension version: version 13.0
- VSCode version:  Visual Studio Code 1.50.1
- OS version: Windows 10

**Steps to reproduce**

1. Started Visual Code Studio
2. All Bookmarks had disappeared
3. Tried to create new bookmark using keyboard shortcut
4. Got error: command 'bookmarks.toggle' not found

**Comment by alefragnani** [comment]
Hi @kalahiri ,

If the error is `command `X` not found`, I would say the extension _didn't activate_, and that's the reason the bookmarks aren't visible.

I would ask you:

* Did you noticed any error message while loading VS Code? 
* Run `Toggle Developer Tools` command and look at `Console` tab. Does it show any error?

If not, I would ask you to reinstall the extension, but doing the full process:

1. Uninstall the extension
2. Close all VS Code instances
3. Restart VS Code _(this full restart will guarantee the extension is totally removed)_
4. Reinstall the extension

If it still does not work, please report any error you see and reinstall a previous version, using the `Install Another Version` command, from the Extensions panel

Hope this helps

**Comment by alefragnani** [comment]
BTW, I would ask you a few details about your installation:

* Which kind of project are you opening? 
  * single folder
  * multi-root workspace
  * remote
* How is defined the `saveBookmarksInProject` setting?

Thank you

**Comment by kalahiri** [comment]
I have opened a single workspace in VSC. I am not familiar with multi-root workspaces, so don't know if that is the case.

I was under the impression the extension was not enabled properly, so I disabled the extension, closed VSC en enabled the extension again. Unfortunately, the error persisted. 

I will follow the steps you mentioned earlier and keep you updated.

**Comment by kalahiri** [comment]
Aha! I opened the console and the extension did not activate. It says:

abstractExtensionService.ts:708 
Activating extension 'alefragnani.Bookmarks' failed: this.context.globalState.setKeysForSync is not a function.

**Comment by alefragnani** [comment]
Gotcha!

There is a "pre-requisite" error on extension's definition. The VS Code minimum requirements should be version 1.51, and not 1.50. This `setKeysForSync` API was released in 1.51.

I'll publish a new version ASAP, to avoid other users from updating on the wrong VS Code version.

In your case, you should update VS Code to a newer version (1.51 or later), or rollback the Bookmarks extension to a previous version. Sorry about that.

Thank you

**Comment by totallytavi** [comment]
Issue replicated successfully in v13.4.2. Here's the error:
```
2023-12-27 23:27:09.347 [error] Activating extension alefragnani.Bookmarks failed due to an error:
2023-12-27 23:27:09.347 [error] Error: Cannot find module 'USER_PROFILE\.vscode\extensions\alefragnani.bookmarks-13.4.2\dist\extension-node.js'
```
